### PR TITLE
223 multiple search projects

### PR DESCRIPTION
**Pull request by kenjinp**
## Changes
- Fix security vulnerability stemming from Parcel-Bundler
- Upgrade to babel 7
- Fix a problem causing morphologies to fail rendering
- Don't create 500 server errors if ES queries come up empty
- refactor some ETL scripts

## Caveats
- For some unknown reason, source-maps are broken with this version of Parcel, but it's the only version combination that works without the security vulnerability. This will make debugging the client difficult.

### CODE REVIEW

File: client/components/MorphologyViewer/index.js [hunk 1]
```diff
@@ -75,11 +75,12 @@ class MorphologyContainer extends React.Component {
   }
   makeVisualizer() {
     let { morphoData } = this.state;
+    console.log({ morphoData });
```
Comment (mdupont) [inline client/components/MorphologyViewer/index.js]: oops

File: package.json [hunk 2]
```diff
@@ -94,11 +97,19 @@
...
   },
   "devDependencies": {
+    "@babel/cli": "^7.1.5",
+    "@babel/core": "^7.1.6",
+    "@babel/node": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.6",
+    "@babel/preset-react": "^7.0.0",
```
Comment (mdupont) [inline package.json]: Do you need to specify all these deps? I thought that just adding preset-react would have it depend on the right stuff, but I may be wrong.

File: package.json [hunk 3]
```diff
@@ -94,11 +97,19 @@
...
+    "@babel/runtime-corejs2": "^7.1.5",
     "@types/jest": "^23.1.6",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (mdupont) [inline package.json]: What is the "bridge" version?

Comment (julienmachon) [reply package.json]: I had to use it on another project. It's due to fact they've changed the "architecture" of babel (with babel-core) and I think this ease the transition between the old version (that some other plugins depends on) and the new stuff.

File: package.json (hunk 2 above)
Comment (kenjinp) [reply package.json]: It seems to be the case, yeah

File: package.json (hunk 3 above)
Comment (kenjinp) [reply package.json]: It turns out it's not necessary :)

File: client/components/MorphologyViewer/index.js (hunk 1 above)
Comment (kenjinp) [reply client/components/MorphologyViewer/index.js]: 😧we should lint for them!

**Pull request merged**